server/config.go

```diff
@@ -99,6 +99,10 @@ type Config struct {
 	EnterpriseGithubStatusEETests string
 	EnterpriseWorkflowName        string
 
+	TranslationsMattermostWebhookURL string
+	TranslationsMattermostMessage    string
+	TranslationsBot                  string
```

Comment:
This field is unused. Any reason for that?

Reply:
Nice catch! Totally messed up there.